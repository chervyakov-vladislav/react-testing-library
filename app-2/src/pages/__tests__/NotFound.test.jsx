import { screen, render } from '@testing-library/react';

import { NotFound } from '../NotFound';

describe('NotFound', () => { 
  it('рендер NotFound', () => {
    render(<NotFound />);

    expect(screen.getByRole('heading')).toMatchSnapshot();
  });
});