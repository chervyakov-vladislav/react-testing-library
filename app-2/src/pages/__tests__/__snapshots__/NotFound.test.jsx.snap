// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound рендер NotFound 1`] = `
<h1>
  Page not found
</h1>
`;
